front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -86,7 +86,7 @@ export function IsolatedSpeedLimitByTagSelector({
           condensed ? 'd-flex align-items-center gap-10' : ''
         }`}
       >
-        <img width="32px" className="mr-2" src={icon} alt="infraIcon" />
+        <img width="32px" src={icon} alt="infraIcon" />
```

Comment:
Alt attribute is wrong here if you could change it, thanks.

Moreover, I'm curious about the hardcoded alt here regardless of the langage. I'm not sure a screen reader would translate this alt if the browser isn't in english. You should ask the UX about that.
```suggestion
        <img width="32px" src={icon} alt="speedometer" />
```